ethcore/engines/authority-round/src/lib.rs

```diff
@@ -1430,6 +1452,46 @@ impl Engine for AuthorityRound {
 		block_reward::apply_block_rewards(&rewards, block, &self.machine)
 	}
 
+	/// Make calls to the randomness and validator set contracts.
```

Comment:
We don't call the `ValidatorSet` contract here, so this comment should be changed.

Reply:
Fixed.